content/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners.md

```diff
@@ -37,6 +37,14 @@ Each CODEOWNERS file assigns the code owners for a single branch in the reposito
 
 For code owners to receive review requests, the CODEOWNERS file must be on the base branch of the pull request. For example, if you assign `@octocat` as the code owner for *.js* files on the `gh-pages` branch of your repository, `@octocat` will receive review requests when a pull request with changes to *.js* files is opened between the head branch and `gh-pages`.
 
+## CODEOWNERS and forks
+
+Asides being used to tell at a glance who is responsible for a given file, CODEOWNERS are also used for triggering review requests for a pull request that modifies an owned file.
```

Comment:
I'm not so sure we need this sentence; I think it's covered in the "About code owners" section, and I think we can keep this section more focused on forks by avoiding that repetition. What do you think? 🙂 

```suggestion
```

Reply:
I agree with dropping it.

Beyond that, I'd suggest avoiding the word `Asides`. You can sometimes use the word `beyond` or `In addition to`.